src/Uno.UI/UI/Xaml/Setter.cs

```diff
@@ -56,12 +56,23 @@ public Setter()
 			}
 			set
 			{
-				ValidateIsSealed();
+				if (!IsMutable)
+				{
+					ValidateIsSealed();
+				}
 
 				_value = value;
 			}
 		}
 
+		/// <summary>
+		/// This should never be called from user code.
+		/// </summary>
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		// Uno docs: As WinUI parses XAML files at runtime, they can set this while it's not public.
+		// In our case, we can't because this needs to be set by XAML generator.
+		public bool IsMutable { get; set; }
```

Comment:
Can we make this an extension method or a helper? This avoids making difficult breaking changes if we even move that field somewhere else.